### PR TITLE
Colours

### PR DESCRIPTION
**Pull request by tduva**
* Changed some colours so they are more easily distuingished. These work better for myself, but I don't know how it would be for others with different or more severe colorblindness (or screens, or whatever). Either way, it should hopefully be better.
* Added a symbol to each colour, toggleable in the options.
* I re-added the original hover effect (changing colour brightness) since the other options didn't really seem that nice. Since this would mostly affect the player while changing colours (viewers would mostly just have to recognize non-hovered colours), the symbols can be enabled if distuingishing colours is an issue.
* Added keycodes for 0-6 on the NumPad, which could be useful.

Edit: I didn't rename the CSS classes for the changed colours since I wasn't sure what to call the colours or whether they are even final. Not sure if it would be better to rename them to something neutral like "colour1" rather than the actual colours. Idk.

### CODE REVIEW

File: bingo.js [hunk 1]
```diff
@@ -435,6 +451,21 @@ function changeColourCount(value)
 	pushNewLocalSetting(COLOUR_COUNT_SETTING_NAME, COLOURCOUNT);
 }
 
+function updateColourSymbols()
+{
+	$(".symbol").css("display", COLOURSYMBOLS == 0 ? "none" : "inline");
```
Comment (rybak) [inline bingo.js:456]: `COLOURSYMBOLS` is always either `true` or `false`. Comparison to zero is not needed.

File: index.html [hunk 2]
```diff
@@ -103,10 +103,11 @@ <h1>Minecraft Bingo</h1>
 				<div class="dropdown-holder">
 					<button class="button options-button dropdown-button" id="options-toggle-button">Options...</button>
 					<div id="options-dropdown-main" class="dropdown">
-						<div class="slider-holder">
+						<div class="slider-holder" title="The number of colours to cycle through when clicking on a square. Can always press (1-6) or (0/q) while hovering over square to select colours.">
```
Comment (rybak) [inline index.html:106]: Having `title` to show tooltips on hover is very nice. Thanks.

Comment (rybak) [reply index.html:106]: It might be a good idea to add `title` to the difficulty slider as well.

**Comment by rybak** [comment]
It seems that `lightning.png` isn't used, so it could be removed.

File: bingo.js (hunk 1 above)
Comment (tduva) [reply bingo.js:456]: I had it as a number before, I guess I missed that one.

**Review by rybak — APPROVED** [review]

**Review by koutsie — APPROVED** [review]
LGTM :ok_hand:

File: bingo.js [hunk 3]
```diff
@@ -435,6 +451,21 @@ function changeColourCount(value)
...
+function updateColourSymbols()
+{
+	$(".symbol").css("display", !COLOURSYMBOLS ? "none" : "inline");
+	const button = document.getElementById("colourSymbols");
+	button.innerHTML = COLOURSYMBOLS ? "Hide Symbols" : "Show Symbols";
+}
+
+function toggleColourSymbols(value)
+{
+	// Invert HIDDEN setting, then update
```
Comment (rybak) [inline bingo.js:463]: I've just noticed that this comment should be updated after copy-pasting from `toggleHidden()`.

**Pull request merged**